todo_list/static/src/main.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module **/
+import { startWebClient } from "@web/start";
+import rpc from "web.rpc";
+
+const {Component} = owl;
+const {onWillStart, useState, useRef} = owl.hooks;
+
+const components = {};
+
+components.Input = class Input extends Component {
+
+        setup() {
+            super.setup();
+            this.state = useState({
+                todo: "",
+            });
+        }
+
+        async addTodo() {
+            await rpc.query({
+                   model: "todo.todo",
+                   method: "create",
+                   args: [[{'name': this.state.todo}]],
+                });
+            this.state.todo = "";
+            this.props.state.output.comp.getTodos();
```

Comment:
Do we really need to invalidate the entire list here? We could simply add this.state.todo